### PR TITLE
17750 - SBC Staff Page View Title and Breadcrumb Update

### PR DESCRIPTION
**Pull request by AbrahamRostampoor**
*Issue #:*
https://github.com/bcgov/entity/issues/17750

*Description of changes:*
Updated page title and breadcrumbs for SBC Staff view of My Business Registry page 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2563-u3cw66hr.web.app

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 1]
```diff
@@ -567,6 +567,9 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   }
 
   get viewTitle (): string {
+    if (this.isSbcStaffAccount) {
+      return 'My Staff Business Registry'
+    }
```
Comment (Jxio) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:572]: reduce the nested ternary conditions, eg:
```
get viewTitle(): string {
  if (this.isSbcStaffAccount || this.isStaffAccount) {
    return 'My Staff Business Registry'
  }
  return 'My Business Registry'
}
```

Comment (AbrahamRostampoor) [reply auth-web/src/components/auth/manage-business/EntityManagement.vue]: Request applied.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2563-u3cw66hr.web.app

**Review by Jxio — APPROVED** [review]
LGTM!

**Pull request merged**